### DISCUSSION TITLE
Get Caniuse to support importing stats

### DISCUSSION THREAD

**Discussion by Lucent** [plausible/analytics#2311] [Anything Else]
[Caniuse.com imports stats from Google Analytics and Simple Analytics](https://caniuse.com/ciu/import). [I have suggested to them using your API to support import from Plausible as well](https://github.com/Fyrd/caniuse/issues/6146). Not sure what kind of cooperation or incentives would be required to make this happen, but I'm happy to contribute to a bounty to get it to happen. It does not appear the backend of caniuse is open source, so I was unable to work on it myself.

**Comment by metmarkosaric** [comment]
"i was imagining Plausible importing the raw caniuse data (from https://github.com/Fyrd/caniuse) so that I can look up some basic support data with one click any time I want"

([feedback from mastodon](https://fosstodon.org/@b0rk@jvns.ca/110383719991453891))

**Comment by merwok** [comment]
This is a great idea. For now, we have to get our stats from plausible and manually create criterion in caniuse to decide whether to use some feature that is not widely available. Exporting a file from plausible that we can feed to caniuse would save that time.
